extends/_a11y.scss

```diff
@@ -1,7 +0,0 @@
-// Hide elements in sake of accessibility
-
-%a11y {
-	position: absolute;
-	top: -32768px;
-	left: -32768px;
-}
```

Comment:
it's redundant (as in visuallyhidden is achieving the same result) and not reliable. values are too high for old firefoxes and if you move many element with this method it is very performance intensive and eventually causes crashes.